cvat/apps/google/README.md

```diff
@@ -0,0 +1,48 @@
+## Google Analytics for CVAT
+
+### Description
+
+This application allows you to integrate Google Analytics with CVAT
+
+### Configuring
+
+  * Set up a Google Analytics (GA) account and create a GA New Property for CVAT
+  * Copy the Tracking ID for the new property
+  * Navigate to `/cvat/cvat/apps/google/static/google/js` and use something like `vi` to edit the `enginePlugin.js` file
+  * Paste your Tracking ID replacing the placeholders `<GA_Tracking_ID_Here>`.  Example code below:
+  
+```
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=<GA_Tracking_ID_Here>"></script>
```

Comment:
@HeffRaleigh , I think that we need to find a way better. It is not good to ask a user to modify code. One possible way is to define a django setting for that (GOOGLE_ANALYTICS_TRACKING_ID) and use it in google/__init__.py to substitute the `<GA_Tracking_ID_Here>` value automatically. What do you think?

Reply:
@nmanovic I like that better and will give it a try.  Might be next week until I can get to it as this week is the close of our quarter/fiscal year and my calendar is booked solid most of the week.  I'll come back here after I give your approach a try and have working code.